ConfigSpace/hyperparameters.pyx

```diff
@@ -1048,13 +1050,14 @@ cdef class NormalFloatHyperparameter(FloatHyperparameter):
     def _sample(self, rs: np.random.RandomState, size: Optional[int] = None
                 ) -> Union[np.ndarray, float]:
 
-        if self.lower == None:
+        sigma = self.sigma
+        if sigma == 0:
+            return self.mu
+        elif self.lower == None:
```

Comment:
This is a nice change in of itself. Depending on how this PR goes, I still think we'd like to pull this in